tools/onnx/onnx2ncnn.cpp

```diff
@@ -3157,6 +3161,22 @@ int main(int argc, char** argv)
             fprintf(pp, " 0=%d", pool);
             fprintf(pp, " 4=%d", global_pool);
         }
+        else if (op == "adaptive_avg_pool2d" || op == "adaptive_max_pool2d")
+        {
+            int pool = 0;
+            if (op == "adaptive_avg_pool2d")
+            {
+                pool = 1;
+            }
+            int adaptive_pooling = 1;
+            const onnx::TensorProto& out_shape_tp = weights[node.input(1)];
+            std::vector<int> out_shape = get_node_attr_from_input_ai(out_shape_tp);
+
+            fprintf(pp, " 0=%d", pool);
+            fprintf(pp, " 7=%d", adaptive_pooling);
+            fprintf(pp, " 17=%d", out_shape[0]);
+            fprintf(pp, " 18=%d", out_shape[1]);
```

Comment:
Is the out_shape hw order or wh order ?

Reply:
The out_shape is  NCHW, so it is hw order.